benchmark/main.py

```diff
@@ -28,9 +28,20 @@ def main(args):
             warnings.warn('contains NaN, skip one trial.')
             continue
 
-        auc.append(eval_roc_auc(y, score))
-        ap.append(eval_average_precision(y, score))
-        rec.append(eval_recall_at_k(y, score, k))
+        auc_val = eval_roc_auc(y, score)
+        auc.append(auc_val)
+
+        ap_val = eval_average_precision(y, score)
+        ap.append(ap_val)
+
+        rec_val = eval_recall_at_k(y, score, k)
+        if isinstance(rec_val, torch.Tensor):
+            rec_val = rec_val.tolist()
+        rec.append(rec_val)
+
+    auc = torch.tensor(auc)
+    ap = torch.tensor(ap)
+    rec = torch.tensor(rec)
```

Comment:
Is that necessary to change L31-L40? Can we simply keep L42-L44?

Reply:
I have checked it. We can simply keep L42-L44. L31-L40 changes are unnecessary. Because at the end of `for loop`, all types of lists will be converted to tensor type. I will change as before as it was.

Reply:
I have checked it. We can simply keep L42-L44. L31-L40 changes are unnecessary. Because at the end of `for loop`, all types of lists will be converted to tensor type. I will change as before as it was.

Reply:
done!